libindy/src/api/mod.rs

```diff
@@ -105,7 +105,7 @@ pub enum ErrorCode
     WalletStorageError = 210,
 
     // Error during encryption-related operations
-    WalletEncryptonError = 211,
+    WalletEncryptionError = 211,
```

Comment:
> WalletEncryptionError 

I suggest to remove it in favor of:

- WalletAccessError if key provided by use is invalid
- ComminInvalidStructure if wallet export file has structure problems

Reply:
Its an old error, used throughout wallet, not added now